rest-service/manager_rest/resource_manager.py

```diff
@@ -2169,6 +2239,7 @@ def _create_deployment_environment(self,
             }
         )
         deployment.create_execution = create_execution
+        deployment.latest_execution = create_execution
```

Comment:
that one is probably not needed, as Alex mentioned in the email chain, create doesn't really need to be included

Reply:
mm, well the reason why I set this here so that I can get the statuses of deployment at creation using `latest_execution_status` thats why I set this here but I think we can avoid that and instead check if the `latest_exeuction` is None then read the execution from the creation instead.

This is the property I used to get the latest execution

```
    @property
    def latest_execution_status(self):
        if not self.latest_execution:
            return None
        return DeploymentState.EXECUTION_STATES_SUMMARY.get(
            self.latest_execution.status
        )
```
so in case the latest_execution` is not set then thats means the deployment is just created which could be re-write like this

```
    @property
    def latest_execution_status(self):
        _execution = self.latest_execution or self.create_execution
        return DeploymentState.EXECUTION_STATES_SUMMARY.get(_execution.status)
        
```